WordPress/Classes/ViewRelated/Tools/SettingsTextViewController.m

```diff
@@ -135,6 +223,10 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
         [self.navigationController popViewControllerAnimated:YES];
     }
 
+    if ([string isEqualToString:@"\n"]) {
```

Comment:
That's the first time I see checking for `\n`. I think the official way is to implement `textFieldShouldReturn:`

> The text field calls this method whenever the user taps the return button. You can use this method to implement any custom behavior when the button is tapped. For example, if you want to dismiss the keyboard when the user taps the return button, your implementation can call the resignFirstResponder method.